docs/en/03_Upgrading/index.md

```diff
@@ -518,6 +518,23 @@ mappings:
   ExplosiveTennisBall: App\Web\Products\ExplosiveTennisBall
 ```
 
+#### Rename Warnings {#rename-warnings}
```

Comment:
That section is about the `add-namespace` command. It would make more sense further down under the "Automatically update namespaced references with the `upgrade` command".